editions/tw5.com/tiddlers/macros/tag-picker_Macro.tid

```diff
@@ -19,7 +19,8 @@ The <<.def tag-picker>> [[macro|Macros]] generates a combination of a text box a
 : <<.from-version 5.3.4>> Defines the target tiddler, which should be manipulated. Defaults to: <<.var currentTiddler>>.
 
 ; tagListFilter
-: <<.from-version 5.3.4>> This parameter defaults to: `[tags[]]` which creates a list of all existing tags. If the tag list should come from a different source the filter should look similar to eg: `[<listSource>get[field-name]enlist-input[]]`. 
+: <<.from-version 5.3.4>> This parameter defaults to: `[tags[]]` which creates a list of all existing tags. If the tag list should come from a different source the filter should look similar to eg: `[<listSource>get[field-name]enlist-input[]]`. See examples.
+: <<.from-version 5.3.6>> This parameter defaults to: `[tags[]sort[]]`. This change allows a custom sort order, since `sort[]` is not hardcoded into the tag-picker macro anymore. 
```

Comment:
@pmario @Jermolene I assume this a typo and it should state 5.3.5 and 5.3.6?

Reply:
Thanks @saqimtiaz I've pushed a fix in 1eb1af25895a0494474e201418cb723e3bd291cc